integration_tests/androidx_test/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,6 @@
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.robolectric.integration.axt">
-    <uses-sdk android:targetSdkVersion="28"/>
```

Comment:
Could you restore related changes? Actually, there is a starter issue to fix this problem, and one folk is trying it. I know it is very simple, but I want to leave it for new contributor without enough experience for open-source project.

Reply:
940f5aeffd2b4efac561e6b647dbedd16b1c8cf6